application/dash_application/views/preqc_wgs.py

```diff
@@ -482,14 +556,17 @@ def init_callbacks(dash_app):
             State(ids['second-sort'], 'value'),
             State(ids['colour-by'], 'value'),
             State(ids['shape-by'], 'value'),
+            State(ids["reads-per-start-point-slider"], 'value'),
+            State(ids["insert-mean-slider"], 'value'),
+            State(ids["passed-filter-reads-slider"], 'value'),
         ]
     )
     def update_pressed(click,
                        runs,
                        first_sort,
                        second_sort,
                        colour_by,
-                       shape_by):
+                       shape_by, total_reads_cutoff, insert_mean_cutoff, rpsp_cutoff):
         if not runs:
             df = pd.DataFrame(columns=WGS_DF.columns)
```

Comment:
It looks like this should be `EMPTY_WGS.columns`, could you make this change?

Reply:
WGS_DF and EMPTY_WGS should have the same columns

Reply:
Aha, when I tested it yesterday and removed runs it still graphed all the data and i thought that was an issue with which df had its columns passed in here. I can't replicate this issue so I'm going to mark this as resolved.